src/algebra/big_operators/basic.lean

```diff
@@ -1721,6 +1721,58 @@ begin
   { rw [finset.sum_insert hat, finset.prod_insert hat, multiset.prod_add, ih] }
 end
 
+lemma disjoint_list_sum_right {a : multiset α} {l : list (multiset α)}
+  (h : ∀ b ∈ l, multiset.disjoint a b) :
+  multiset.disjoint a l.sum :=
+begin
+  classical,
+  induction l with b bs ih,
+  { exact (zero_disjoint _).symm },
+  { rw [list.sum_cons, disjoint_add_right],
+    exact ⟨h _ (list.mem_cons_self _ _), ih $ λ b hb, h _ $ list.mem_cons_of_mem _ hb⟩ }
+end
+
+lemma disjoint_list_sum_left {a : multiset α} {l : list (multiset α)}
+  (h : ∀ b ∈ l, multiset.disjoint b a) :
+  multiset.disjoint l.sum a := (disjoint_list_sum_right $ λ b hb, disjoint.symm (h b hb)).symm
+
+lemma disjoint_sum_right {a : multiset α} {i : multiset (multiset α)} :
+  (∀ b ∈ i, multiset.disjoint a b) → multiset.disjoint a i.sum :=
+by { classical, exact (quotient.induction_on i $ λ l h, begin
+     rw [quot_mk_to_coe, multiset.coe_sum],
+     exact disjoint_list_sum_right h,
+   end) }
```

Comment:
```suggestion
quotient.induction_on i $ λ l h, begin
   rw [quot_mk_to_coe, multiset.coe_sum],
   exact disjoint_list_sum_right h,
 end
```

Reply:
Note that you can test the linter by writing `#lint`, you don't need to wait hours.

Adding `classical` doesn't fix the problem that you're running into; the problem is that you need to put these lemmas above the `variables [decidable_eq α]` above, because your proof doesn't use that argument.

Reply:
I see, thanks for the tip.